ComputerInterface/BaseGameInterface.cs

```diff
@@ -49,21 +46,36 @@ public static void SetName(string name)
 
             if (!GorillaComputer.instance.CheckAutoBanListForName(name)) return;
 
-            PhotonNetwork.LocalPlayer.NickName = name; 
+            PhotonNetwork.LocalPlayer.NickName = name;
             GorillaComputer.instance.offlineVRRigNametagText.text = name;
             GorillaComputer.instance.savedName = name;
             PlayerPrefs.SetString("playerName", name);
             PlayerPrefs.Save();
-            
+
             /* Player's name is not updating on change */
             if (PhotonNetwork.InRoom)
             {
                 GetColor(out var r, out var g, out var b);
-                GorillaTagger.Instance.myVRRig.photonView.RPC("InitializeNoobMaterial", RpcTarget.All, (object)r, (object)g, (object)b);
+                InitializeNoobMaterial(r, g, b);
             }
             /* Player's name is not updating on change */
         }
 
+        public static void InitializeNoobMaterial(Color colour) => _InitializeNoobMaterial(colour);
+
+        public static void InitializeNoobMaterial(float r, float g, float b) => _InitializeNoobMaterial(new Color(r, g, b));
+
+        private static void _InitializeNoobMaterial(Color colour)
```

Comment:
Why is this hidden as a private method when InitializeNoobMaterial(Color colour)  is exposed